applications/skyepass.md

```diff
@@ -0,0 +1,242 @@
+
+```javascript
+{
+	message: 'a decrypted piece of the original file.......',
+    signer: 'a publickey',
+    signature: 'a signature of the signer onto the message'
+}
+```
+
+
+### Ecosystem Fit 
+
+SkyeKiwi will be a unique component in the ecosystem of a storage solution network. We are providing an off-chain encryption and metadata handling toolkit combined with on-chain NFT-like smart contract solution to achieve decentralized secret sharing. Such protocol shall unlock the potential for permission NFTs or decentralized and verifiable contract signature. 
+
+## Team
+
+#### Team members
+
+* Song Zhou (Full stack developer) https://github.com/RoyTimes
+* Zoe Sun (Lead Designer)
+* ... More to be hired 
+
+#### Contact
+* **Contact Name:** Song Zhou (song.zhou@skye.kiwi)
+
+#### Legal Structure 
+* No legal entity yet
+
+#### Team's experience
+Besides private work for companies that cannot be shared, a password manager based on the SkyeKiwi protocol SkyePass is now officially supported by the Web3 Foundation Grant Program
+
+#### Team Code Repos
+* https://github.com/skyekiwi/
+
+## Development Roadmap
+
+### Overview
+* **Total Estimated Duration:** 8 weeks
+* **Full-time equivalent (FTE):**  2.5 FTE 
+* **Total Costs:** $22,400
+
+### Milestone 1 — Core Protocol
+* **Estimated Duration:** 4 Weeks
+
+* **FTE:**  2
+
+* **Costs:** \$11,200 (2 FTE * 35 Hours per week * 4 Weeks * $40Hourly Wage.)
+
+    
+  
+  Generally, we will build the core protocol libraries as described above, and provide a sample smart contract for developer team to use. We will provide a brief deployment guide but will not deploy the smart contract to any blockchains. It will be totally up to the user's discretion as of how to use it. 
+  
+  
+
+| Number | Deliverable           | Specification                                                |
+| ------ | --------------------- | ------------------------------------------------------------ |
+| 0a.    | License               | Apache 2.0 on all repos                                      |
+| 0b.    | Documentations        | A guideline of how to run and test all functionalities described below. |
+| 1.     | Sample Smart Contract | The core smart contract that stores IPFS hash, generate unique vault ID and implement access management. <br/>We are using ink! and the smart contract development suite maintained and developed by [Patract Labs](https://patract.io/) for developing environment, unit testing and deployment. |
+| 2.     | Protocol Library      | 1. 3 sample encryption schema as described above and tooling to build customized encryption schema. <br/>2. Core metadata handler that takes in a file and the encryption schema to generate a sharable metadata file. <br/>3. Use a sample smart contract to facilitate authorization management and file updating. <br/>4. Persistent storage with Crust Network APIs<br/>5. Unit tests and integration test w/Chia |
```

Comment:
what's this *chia* means? Is that the blockchain chia or a testing framework?

Reply:
the testing framework on node.js